tests/unit/onyxCacheTest.js

```diff
@@ -400,8 +400,8 @@ describe('Onyx', () => {
 
     describe('Onyx with Cache', () => {
         let Onyx;
+        let LocalForageMock;
```

Comment:
Technically if we have mocks for each storage provider we can re-run the test suite with another storage provider mock

But usually this has no benefit, because in either case it's just a mock - not the real thing
A test does not depend on a storage provider (unless it tests the actual provider), but on the assumption that if storage works "correctly" it results in the expected result 
- `getItem` returns a Promise with an item
- `setItem` returns a Promise

the test is about whether we use the result of `getItem` correctly
the test is whether we call `setItem` with expected parameters
the specific storage provider doesn't matter

Reply:
I think it would also make sense, to remove the `LocalForageProviderTest.js` tests, since we're not really testing `localForage`, only the mock

Reply:
> I think it would also make sense, to remove the `LocalForageProviderTest.js` tests, since we're not really testing `localForage`, only the mock

Not exactly - the test verifies we're using the underlying library `localforage` as intended 
For example a test verifies that if we call `StorageProvider.multiSet(pairs)` then for each pair we call `localforage.setItem` with the expected `key` and `value` 

So the test is would `multiSet` call `setItem` correctly for each pair - we don't have to use the real localforage library to test that